### DISCUSSION TITLE
Надо добавить статьи на вики

### DISCUSSION THREAD

**Discussion by Bulat-Ziganshin** [tarantool/tarantool#6767] [General]
На https://github.com/tarantool/tarantool/wiki/Resources можно добавить ещё несколько статей с хабра:

https://habr.com/ru/company/vk/blog/562192/
https://habr.com/ru/company/vk/blog/533420/
https://habr.com/ru/company/vk/blog/525484/

собственно, я их выловил на https://habr.com/ru/hub/tarantool/ - может, просто доабвить на него ссылку?

**Comment by Totktonada** [comment]
Done: https://github.com/tarantool/tarantool/wiki/Resources/_compare/03fc0b0d577939e879c7e87b6bc9adbd66280f76...ffc62d81f1ad3a1f2081b1448e25a976c5f61a58.
